### ISSUE TITLE
Invulling publieke organisatie verruimen

### ISSUE THREAD

**Issue by marleenvd** [Informatievlaanderen/OSLO-Discussion#269]
**Model(len) waarop de issue van toepassing is:**
DCAT-AP-VL

**Omschrijving van het probleem:**
Indien de definitie van publieke organisatie correct wordt toegepast kan binnen de Vlaamse overheid geen onderscheid gemaakt worden tussen de verschillende entiteiten. Dit zijn immers entiteiten zonder rechtspersoonlijkheid (= in strijd met voorwaarde (b) zoals verwoord in de toelichting bij "publieke organisatie"). In de KBO vallen die entiteiten immers onder één en hetzelfde ondernemingsnummer: m.n. de onderneming ‘Ministeries van de Vlaamse Gemeenschap’ (MVG) - 0316.380.841. Door oneigenlijk gebruik van KBO worden entiteiten echter uiteindelijk wel zichtbaar in KBO ([https://overheid.vlaanderen.be/de-vlaamse-overheid-in-kbo]).
Anderzijds zijn er datasets die ontstaan door data, die beheerd wordt door meerdere entiteiten, samen te ontsluiten. Dit is het geval binnen Databank Ondergrond Vlaanderen. Er wordt gewerkt binnen een formeel samenwerkingsverband. In de definitie van Publieke organisatie staat te lezen " ... verenigingen gevormd door een of meer ...". Deze bepaling lijkt mij voldoende basis zijn om ruimer te gaan dan de 66 entiteitscodes die op dit moment worden weerhouden.  Wellicht is deze situatie ook van toepassing voor andere datasets ? Ter info: binnen Wegwijs bestaat  voor Ondersteunend Centrum Databank Ondergrond Vlaanderen een organisatie met code OVO027735. 

**Omschrijving van een mogelijke oplossing:**
Definitie van publieke organisatie moet op zich al ruimer worden geïnterpreteerd om binnen de VO onderscheid te kunnen maken. Verdere verruiming nodig voor datasets die via "netwerkorganisatie" ontsloten worden.

**Comment by bertvannuffelen** [comment]
Dag Marleen,

De definitie is overgenomen van het applicatie profiel organisatie-basis (https://data.vlaanderen.be/doc/applicatieprofiel/organisatie-basis/#Publieke%20Organisatie). 
Als je opmerkingen hebt over de definitie dan is het best om contact op te nemen met Thomas D'Haenens. 

Ivm DCAT-AP Vlaanderen: we hadden dit zo opgenomen in de figuur om de relatie met het organisatieregister visueel te maken. We gaan dit nu enkel opnemen als een gebruiksnota.

Daarbij toch nog een opmerking op je lezing van de figuur. Er staat dat een PubliekeOrganisatie een subklasse is van Agent. Dat betekent dus dat elke PubliekeOrganisatie een Agent is, en deze entiteiten kunnen dus worden opgegeven als een uitgever. Echter niet alle Agenten zijn PubliekeOrganisaties. Ook deze laatste kunnen dus uitgever zijn. Dus vanuit DCAT-AP Vlaanderen is er geen nood aan een verruiming van de definitie van PubliekeOrganisatie.  
Met het verwijderen van deze klasse is het dan ook duidelijk dat dit geen herdefinitie is van een Publieke Organisatie.

mvg,
Bert

**Comment by dhaeneth** [comment]
Marleen,

Ik denk dat je hier een foute interpretatie maakt. 
Je verwijst naar een voorwaarde bij de definitie die zou eisen dat de publieke organisatie rechtspersoonlijkheid zou hebben? Kun je me tonen waar je die precies terugvindt? Dat betwijfel ik eigenlijk wel.
Je verwijst ook naar de relatie tussen de Vlaamse overheden en de KBO en maakt melding van 66 entiteiten (ik vermoed dat je daarmee de EVA's en IVA's met rechtspersoonlijkheid bedoelt?). Er is idd een overlap tussen de VO en de KBO mbt. relevante organisaties, maar dit is enkel een overlap; je mag dat niet gelijk trekken.

De hele doelstelling van het semantische model 'Publieke Organisatie' en de toepassing daarvan in bvb. het organisatieregister is de correcte éénduidige identificatie van organisaties die géén rechtspersoonlijkheid hebben, maar wel een functie vervullen in een bepaald proces. 

Dus het correct toepassen van de definitie 'Publieke Organisatie' leent er zich net wél toe om ook bvb. DOV te identificeren als een publieke organisatie (ic. een samenwerkingsverband tussen verschillende andere organisaties). 
Merk op dat 'Publieke Organisatie' een subklasse is van het concept 'Organisatie' dat ook géén notie kent van rechtspersoonlijkheid. Een andere subklasse van 'Organisatie' is 'Formele Organisatie' die op zijn beurt als subklasse heeft 'Geregistreerde Organisatie' (zie het schema op https://data.vlaanderen.be/doc/applicatieprofiel/organisatie-basis/). Het is op dat niveau 'Geregistreerde Organisatie' dat we semantisch de KBO terugvinden. Merk ook op dat de subklassen van Organisatie niet mutueel exclusief zijn : iets kan dus én een Publieke Organisatie én een Geregistreerde Organisatie zijn (bvb. de Vlaamse Waterweg die rsp. een EVA van publiek recht én een NV is).

**Comment by marleenvd** [comment]
Ik had de voorwaarde m.b.t. rechtspersoonlijkheid afgeleid uit de toelichting op https://data.vlaanderen.be/doc/applicatieprofiel/DCAT-AP-VL/#Publieke%20Organisatie

**Comment by dhaeneth** [comment]
Ja.. begrijpelijke verwarring. Ik was ook niet in detail betrokken bij de uitwerking van dat vocabularium.
Bij die gebruiksaanwijzing staan er verschillende zaken samen : 
- we vinden er enerzijds een definitie van 'openbaar lichaam' : 'de staat, zijn territoriale lichamen, publiekrechtelijke instellingen, en verenigingen gevormd door een of meer van deze lichamen of een of meer van deze publiekrechtelijke instellingen' (lichaam is hierbij een vertaling en is te lezen als "organisatie")
- dan wordt er ook een definitie gegeven van 'publiekrechtelijke instelling'  (die conformeert aan definities gehanteerd binnen de afbakening van de Europese publieke sector). 

In de definitie van publiekrechtelijke instelling is rechtspersoonlijkheid een vereiste. Maar een openbaar lichaam (~een publieke organisatie) telt ook de staat, met haar onderverdelingen (bvb. agentschappen zonder rp), territoriale verenigingen (gemeenten, provincies, gewest) en verenigingen gecreëerd door voorgaande (dus bvb. DOV als vereniging gecreëerd door ...).

Dus de notie rechtspersoonlijkheid slaat enkel op het concept 'publiekrechtelijke instelling'.

**Comment by bertvannuffelen** [comment]
Ik denk dat dit nu verduidelijked is. @marleenvd mag ik dit sluiten?

**Comment by marleenvd** [comment]
OK. Dit mag gesloten worden.
